### ISSUE TITLE
Provide the contributor with submission details, how to cite, how to find analysis results, how to share their submission

### ISSUE THREAD

**Issue by morrisonnorman** [HumanCellAtlas/ingest-central#86]
As an HCA-DCP wrangler I want to provide the contributor with submission details, how to cite, how to find analysis results, how to share their submission.

Migrated from JIRA HCA-1128

**Comment by hewgreen** [comment]
NB for now this would be wrangler led. To break this down our requirements are:

Submission details:
Related to ticket #240
Also, Rolando has already provided some metrics per dataset. Ideally these would be more easily viewable in the UI.

How share their submission:
URL to browser for the project upon completion of submission.

How to cite:
It is unclear what this looks like. Requires input from PMs.

How to find analysis results:
see #240

**Comment by justincc** [comment]
As this is wrangler led I'm going to remove myself for now whilst there are no dev tasks for GA

**Comment by hewgreen** [comment]
@lauraclarke @morrisonnorman Can this ticket be closed?

**Comment by lauraclarke** [comment]
Can we do this?

**Comment by hewgreen** [comment]
To a degree.

**Comment by lauraclarke** [comment]
If it is sufficient for MVP then it can be closed. I don't know how we are logging features which need improvement @morrisonnorman @justincc ? but I suspect this should go on that list?

**Comment by malloryfreeberg** [comment]
I don't think we have the "how to cite" part, unless we are OK telling contributors they can "cite" using the URL, which I don't suggest given I have no idea how stable that will be. Similarly, we can only give directions on how to "share" their submission via the URL.

I think @morrisonnorman suggested planning features using post-it notes on a paper print-out of the user journey map @gabsie made.
